actions/pack_action.go

```diff
@@ -34,8 +37,19 @@ func (pf *PackAction) Run(context *debos.DebosContext) error {
 	pf.LogStart()
 	outfile := path.Join(context.Artifactdir, pf.File)
 
+	var tarOpt = "cf"
```

Comment:
this could break recipes since you are changing the default parameter of `Compression` from `gz`=>`none`.
Maybe you should set the default value for `Compression` to `gz` and handle `none` under `tarOpts`

Reply:
The `Compression` parameter was under the mandatory section so it's not a break from a documentation perspective :stuck_out_tongue:

Reply:
nevertheless, there are some recipes i've worked on which don't include the `compression` argument for this action and assume `tar.gz`

Reply:
Okay, now defaulting to gz and added `none` parameter.